src/style.css

```diff
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(../src/images/star-active.svg);
+  background-position: contain;
+  background-repeat: no-repeat;
+}
+
+.stars__stars:not(:last-child) {
+  margin-right: 0;
+}
```

Comment:
you don't have `stars__stars` class in your html and this implementation seems not correct

Reply:
I'm sorry, but I don't understand what needs to be fixed here (